### PR TITLE
Day3

### PR DESCRIPTION
**Pull request by susird**
This is finally the pr of 3th day for 30 Days of JavaScript :sparkle: :sparkles:

### CODE REVIEW

File: Day3/excercises_day3.js [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  let ofAge = 2003;
+  let yearsOfAge = birthYear - ofAge;
+  let age = today - birthYear;
+  if (birthYear < 18) {
+    alert(`You will be allowed to drive after ${yearsOfAge} years`);
+  } else {
+    alert(`You are ${age} old enough to drive`);
+  }
+}
+//console.log(driveAge())
```
Comment (anyruizd) [inline Day3/excercises_day3.js]: do you need this here?

File: Day3/excercises_day3.js [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+//6.Slope is (m = y2-y1/x2-x1). Find the slope between point (2, 2) and point(6,10)
+function calcSlope(coordinates) {
+  const { x1, x2, y1, y2 } = coordinates;
+  return (y2 - y1) / (x2 - x1);
+}
+//console.log(calcSlope({x1: 2, y1: 2, x2: 6, y2: 10}))
+
+//7.Compare the slope of above two questions.
+//console.log(slopeCalculate >= calcSlope)
```
Comment (anyruizd) [inline Day3/excercises_day3.js]: what about this here? and the spaces below?

File: Day3/excercises_day3.js [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+console.log(typeof city)
+console.log(typeof age)
+console.log(typeof isMarried)
+console.log(typeof year)
+
+//2. Check if type of '10' is equal to 10
+console.log('10' === 10)
+
+//3. Check if parseInt('9.8') is equal to 10
+console.log(parseInt('9.8') == 10)
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: always try to check for type, so use `===`

File: Day3/excercises_day3.js [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+//4. Boolean value is either true or false.
+//i. Write three JavaScript statement which provide truthy value.
+console.log(3 === 3)
+console.log('potato' != 'minino')
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: `'potato' !== 'minino'`

File: Day3/excercises_day3.js [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+
+
+//8. Calculate the value of y (y = x^2 + 6x + 9). Try to use different x values and figure out at what x value y is 0
+function calculateY() {
+  let range = [-8, -7, -6, -5, -6, -5, -4, -3, -2, -1, 0];
+  return range.map(x => Math.pow(x, 2) + 6 * x + 9)
+}
+console.log(calculateY())
+
+//9. Writ a script that prompt a user to enter hours and rate per hour. Calculate pay of the person?
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: Write

File: Day3/excercises_day3.js [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+//What is the minutes now?
+console.log(now.getMinutes())
+//Find out the numbers of seconds elapsed from January 1, 1970 to now.
+console.log(now.getTime())
+
+//Level 2
+/*1.Write a script that prompt the user to enter base and height of the triangle
+ and calculate an area of a triangle (area = 0.5 x b x h).*/
+function triangleArea() {
+  let base = prompt('Insert base here',);
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: why `let` ?

File: Day3/excercises_day3.js [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+    alert(`Your name is short`);
+  }
+}
+//console.log(sizeLength())
+
+//11. Compare your first name length and your family name length and you should get this output.
+function compareLength() {
+  let yourName = prompt('Insert your name here',);
+  let familyName = prompt('Insert your family name here',);
+  if (yourName.length > familyName.length) {
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: what happen if `yourName.length === familyName.length` ?

File: Day3/excercises_day3.js [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+  } else {
+    alert(`You are ${age} old enough to drive`);
+  }
+}
+//console.log(driveAge())
+
+/*14.Write a script that prompt the user to enter number of years. Calculate the number 
+of seconds a person can live. Assume some one lives just hundred years*/
+function secondsLive() {
+  let birthYear = prompt('Enter number of yours you live',);
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: let age= prompt(Enter your age');

File: Day3/excercises_day3.js [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+  let seccondsYear = 31536000;
+  let calculate = birthYear * seccondsYear;
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: calculateSecondsLeft = 100 * seccondsYear - age * seccondsYear;

File: Day3/excercises_day3.js [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+  }
+}
+function twoDigitsMin() {
+  let dateMinutes = now.getMinutes();
+  if (dateMinutes < 10) {
+    return min = "0" + dateMinutes
+  } else {
+    return dateMinutes
+  }
+}
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: why did you not use a function to make this?

File: Day3/excercises_day3.js [hunk 11]
```diff
@@ -0,0 +1,267 @@
...
+
+function twoDigitsHour() {
+  let dateHours = now.getHours();
+  if (dateHours < 10) {
+    return hour = "0" + dateHours
+  } else {
+    return dateHours
+  }
+}
+function twoDigitsMin() {
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: those are the same function:
``` 
const formatTwoDigits = (numberToFormat) => numberToFormat < 10 ? `0${numberToFormat}` : String(numberToFormat);
const dateMinutes = now.getMinutes()
const dateHours = now.getHours();
formatTwoDigits(dateMinutes);
formatTwoDigits(dateHours);
```

File: Day3/index.html [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script src="./excercises_day3.js"></script>
+</body>
+</html>
```
Comment (josepplloo) [inline Day3/index.html]: Insert new line

File: Day3/excercises_day3.js [hunk 13]
```diff
@@ -0,0 +1,267 @@
...
+  let dateMinutes = now.getMinutes();
+  if (dateMinutes < 10) {
+    return min = "0" + dateMinutes
+  } else {
+    return dateMinutes
+  }
+}
+console.log(`${dateYear}-${dateMonth}-${dateDay}` + " " + twoDigitsHour() + ":" + twoDigitsMin())
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: insert New line

**Review by josepplloo — COMMENTED** [review]
nice work! 
We need to make a deep review of the concepts, you will start to understand more and more when you face more challenges don't worry, the beginning is always overwhelming

File: Day3/excercises_day3.js (hunk 9 above)
Comment (susird) [reply Day3/excercises_day3.js]: Change birthYear

File: Day3/excercises_day3.js [hunk 14]
```diff
@@ -0,0 +1,258 @@
+// Level 1
+/*1. Declare firstName, lastName, country, city, age, isMarried, year variable and assign value to it and use the
+ typeof operator to check different data types.*/
+let firstName = 'Susana'
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: why not `const`?

File: Day3/excercises_day3.js [hunk 15]
```diff
@@ -0,0 +1,258 @@
...
+//Find out the numbers of seconds elapsed from January 1, 1970 to now.
+console.log(now.getTime())
+
+//Level 2
+/*1.Write a script that prompt the user to enter base and height of the triangle
+ and calculate an area of a triangle (area = 0.5 x b x h).*/
+function triangleArea() {
+  const base = prompt('Insert base here',);
+  const height = prompt('Insert height gere',);
+  let area = ((0.5) * base * height);
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: const

File: Day3/excercises_day3.js [hunk 16]
```diff
@@ -0,0 +1,258 @@
...
+  const circArea = (Math.PI * circRadius * circRadius);
+  const circumf = ((2) * Math.PI * circRadius);
+  alert(`The area of the circle is  ${circArea}`);
+  alert(`The circumference of the circle is  ${circumf}`)
+}
+//console.log(circleArea())
+
+//5.Calculate the slope, x-intercept and y-intercept of y = 2x -2
+function slopeCalculate() {
+  let x = prompt('Insert the intercept here',);
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: idem

File: Day3/excercises_day3.js [hunk 17]
```diff
@@ -0,0 +1,258 @@
...
+function calculateY() {
+  const range = [-8, -7, -6, -5, -6, -5, -4, -3, -2, -1, 0];
+  return range.map(x => Math.pow(x, 2) + 6 * x + 9)
+}
+console.log(calculateY())
+
+//9. Write a script that prompt a user to enter hours and rate per hour. Calculate pay of the person?
+function pay() {
+  const hours = prompt('Insert hours here',);
+  const rate = prompt(`Insert rate per hour here`,);
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: no trailing comma

Comment (josepplloo) [reply Day3/excercises_day3.js]: if  `  necessary here?

File: Day3/excercises_day3.js [hunk 18]
```diff
@@ -0,0 +1,258 @@
...
+  const familyName = prompt('Insert your family name here',);
+  if (yourName.length >= familyName.length) {
+    alert(`Your firs name, ${yourName} is longer than your family name, ${familyName}`);
+  } else {
+    alert(`Your family name, ${familyName} is longer than your firs name, ${yourName}`);
+  }
+}
+//console.log(compareLength())
+
+//12. Declare two variables myAge and yourAge and assign them initial values and myAge and yourAge.
```
Comment (josepplloo) [inline Day3/excercises_day3.js:196]: what a weird problem 😖

File: Day3/index.html (hunk 12 above)
Comment (josepplloo) [inline Day3/index.html]: your need to insert an empty line to the end of the files to avoid compilation problems

**Review by josepplloo — CHANGES_REQUESTED** [review]
Fix all the code styles problem and youse would merge this PR! 🥇

File: Day3/excercises_day3.js [hunk 19]
```diff
@@ -0,0 +1,258 @@
...
+const dateDay = now.getDay()
+
+function twoDigitsHour() {
+  const formatTwoDigits = (numberToFormat) => numberToFormat < 10 ? `0${numberToFormat}` : String(numberToFormat)
+  const dateHours = now.getHours();
+  const dateMinutes = now.getMinutes();
+  formatTwoDigits(dateHours);
+  formatTwoDigits(dateMinutes);
+}
+console.log(`${dateYear}-${dateMonth}-${dateDay}` + " " + twoDigitsHour() + ":" + twoDigitsHour())
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: twoDigitsHour does not return anything 🤷

File: Day3/excercises_day3.js [hunk 20]
```diff
@@ -0,0 +1,258 @@
...
+}
+//console.log(pay())
+
+//10.If the length of your name is greater than 7 say, your name is long else say your name is short.
+function sizeLength() {
+  const name = prompt('Insert your name here');
+  if (name.length > 7) {
+    alert(`Your name is long`);
+  } else {
+    alert(`Your name is short`);
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: ` -> '

File: Day3/excercises_day3.js [hunk 21]
```diff
@@ -0,0 +1,258 @@
...
+  alert(`The area of the rectangle is  ${recArea}`);
+  alert(`The perimeter of the rectangle is ${recperimeter}`)
+}
+//console.log(rectangle())
+
+/*Get radius using prompt and calculate the area of a circle (area = pi x r x r) 
+and circumference of a circle(c = 2 x pi x r) where pi = 3.14.*/
+
+function circleArea() {
+  const circRadius = prompt('Insert radius here');
```
Comment (josepplloo) [inline Day3/excercises_day3.js]: change the name to `radius`, and try not to use abbreviations. 
only in special cases eg https://www.acronymfinder.com/Circumference-(CIRCUM).html

**Review by josepplloo — CHANGES_REQUESTED** [review]
Only a few comments, 💅

File: Day3/excercises_day3.js (hunk 20 above)
Comment (anyruizd) [reply Day3/excercises_day3.js]: @susird you don't need to use ` here, use ' instead. Back ticks are only needed when you need to interpolate a variable.

File: Day3/excercises_day3.js [hunk 22]
```diff
@@ -0,0 +1,258 @@
...
+  const circum = ((2) * Math.PI * radius);
+  alert(`The area of the circle is  ${circleArea}`);
+  alert(`The circumference of the circle is  ${circum}`)
+}
+circleArea()
+
+//5.Calculate the slope, x-intercept and y-intercept of y = 2x -2
+function slopeCalculate() {
+  let x = prompt('Insert the intercept here');
+  let y = ((2 * x) - 2);
```
Comment (anyruizd) [inline Day3/excercises_day3.js]: Don't see any reason for you to be using `let` here.

**Review by anyruizd — APPROVED** [review]

**Review by josepplloo — APPROVED** [review]
🥇

**Pull request merged**